src/rt/sched/schedulerthread.h

```diff
@@ -105,6 +105,10 @@ namespace verona::rt
     /// cleared before scheduler sleep, or in some stages of the LD protocol.
     ObjectMap<T*> mute_set;
 
+    // The set of poller cowns. These cowns are notified periodically to handle
+    // IO.
+    ObjectMap<Object*> pollers;
```

Comment:
Does this keep the pollers live?  Is there a strong reference on the poller?

If the context does not keep a reference to a poller can it be collected.  I am concerned it can.